tendrl/commons/utils/log_utils.py

```diff
@@ -0,0 +1,31 @@
+""" Handles logging functionality. """
+import sys
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+
+
+def log(log_priority,publisher_id,**log_payload):
+    """
+    Function used for logging errors/output/info.
+    Args:
+        log_priority : Priority of the Log Message (error/info)
+        publisher_id : Id of publisher (mandatory)
+        log_payload  : Payload can contain parameters like message that is to be logged
+    """
+    try:
+        if publisher_id != None:
+            Event(
+                Message(
+                    priority=log_priority,
+                    publisher=publisher_id,
+                    payload=log_payload
+                )
+            )
+        else:
+            raise Exception('Publisher Id not found')
+    except:
+            message = log_payload.get("message","Meesage Not Found") or log_payload.get("Message","Meesage Not Found")
```

Comment:
log_payload.get() has been used in the except clause where the message is supposed to be printed, I have given this particular condition so that in case someone misses out the message then this function should not throw error